AndelaOSP/art-dashboard
# #159521527 Add asset filter button

## What does this PR do?
Adds asset filter button and component

## Description of Task to be completed?
- Create filter button
- Create filter component
- Make filter button toggle

## How should this be manually tested?
On the assets page click on the filter button which can be toggled. 
The filter component should toggle on and off when you click on filter in the user list page.

## What are the relevant pivotal tracker stories?
[#159521527](https://www.pivotaltracker.com/story/show/159521527)

## Any background context you want to add?
None

## Screenshots
<img width="1270" alt="screen shot 2018-08-08 at 3 57 08 pm" src="https://user-images.githubusercontent.com/5074111/43838345-c2a60cc8-9b23-11e8-93f0-33e62cda63cb.png">



<img width="1269" alt="screen shot 2018-08-08 at 3 55 16 pm" src="https://user-images.githubusercontent.com/5074111/43838248-83a8b796-9b23-11e8-853b-54a371487a4e.png">
